apps/address-service/domains/address/schema/Address.js

```diff
@@ -70,6 +69,16 @@ const Address = new GQLListSchema('Address', {
             },
         },
     },
+    kmigratorOptions: {
+        constraints: [
+            {
+                type: 'models.UniqueConstraint',
+                fields: ['key'],
+                condition: 'Q(deletedAt__isnull=True)',
+                name: 'address_key_unique_key',
+            },
+        ],
+    },
```

Comment:
How it's related to this PR?

Reply:
When the address was changed, it's possible that a new Address model was created. I need to remove it and update another Address model, which was used to create property in a condo. These two models have the same `key` field.
There was a restriction `isUnique: true`, but I don't want to restrict deleted models.
The mutation deletes one model and passes its data into another model created earlier.